template/tailwind.config.js

```diff
@@ -0,0 +1,25 @@
+const merge = require('lodash.merge');
+const plugin = require('tailwindcss/plugin');
+const config = require('@studiometa/tailwind-config');
+
+/**
+ * TailwindCSS Configuration File
+ *
+ * @docs    https://tailwindcss.com/docs/configuration
+ * @default https://github.com/studiometa/tailwind-config/blob/develop/src/index.js
+ */
+module.exports = merge(config, {
+  // Extends the default Studio Meta Tailwind configuration here...
+  plugins: [],
+  theme: {},
+  purge: {
+    // Learn more on https://tailwindcss.com/docs/controlling-file-size/#removing-unused-css
+    enabled: process.env.NODE_ENV === 'production',
+    content: [
+      './web/wp-content/themes/<%= name %>/src/js/**/*.js',
+      './web/wp-content/themes/<%= name %>/src/js/**/*.vue',
+      './web/wp-content/themes/<%= name %>/**/*.tpl',
```

Comment:
I think we can remove `*.tpl` from this list, but maybe we should add `*.php` files ?

Reply:
I think it should be `<%= slug %>` also in this path no ?

Reply:
Not sure if adding php files is useful since classes should be used in the view (twig) ?

Reply:
yes, probably better to leave it like this, and add php files if a project need it